platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/Transform.java

```diff
@@ -283,54 +267,54 @@ void setBearing(double bearing, float focalX, float focalY, long duration) {
   //
 
   LatLng getLatLng() {
-    return mapView.getLatLng();
+    return nativeMapView.getLatLng();
   }
 
   //
   // Pitch / Tilt
   //
 
   double getTilt() {
-    return mapView.getPitch();
+    return nativeMapView.getPitch();
   }
 
   void setTilt(Double pitch) {
     markerViewManager.setTilt(pitch.floatValue());
-    mapView.setPitch(pitch, 0);
+    nativeMapView.setPitch(pitch, 0);
   }
 
   //
   // Center coordinate
   //
 
   LatLng getCenterCoordinate() {
-    return mapView.getLatLng();
+    return nativeMapView.getLatLng();
   }
 
   void setCenterCoordinate(LatLng centerCoordinate) {
-    mapView.setLatLng(centerCoordinate);
+    nativeMapView.setLatLng(centerCoordinate);
   }
 
   void setGestureInProgress(boolean gestureInProgress) {
-    mapView.setGestureInProgress(gestureInProgress);
+    nativeMapView.setGestureInProgress(gestureInProgress);
     if (!gestureInProgress) {
       invalidateCameraPosition();
     }
   }
 
   void moveBy(double offsetX, double offsetY, long duration) {
     if (duration > 0) {
-      mapView.addOnMapChangedListener(new MapView.OnMapChangedListener() {
+      mapView.addOnCameraDidChangeListener(new MapView.OnCameraDidChangeListener() {
```

Comment:
This method looks like it can allocate a lot of new anonymous class objects, especially during a move gesture. Maybe we can rework it to use local field instead?